packages/server/src/fhir/repo.ts

```diff
@@ -257,6 +258,8 @@ export class Repository extends BaseRepository implements FhirRepository {
 
     const resource = JSON.parse(rows[0].content as string) as T;
     await setCacheEntry(resource);
+    // TODO(ThatOneBro 01 Dec 23): Check if we can do this in the background
+    // setCacheEntry(resource).catch(console.error);
```

Comment:
Interesting idea... 

It should be background-able / parallel-izable within the scope of the "read" operation.  I think we would still need to `await` before the operation could be considered "complete" though.  To do that would require some refactoring...  you'd basically need some representation of "promises that need to be settled before the operation is done".

Reply:
Yeah that makes sense, maybe I can take a look at doing that...

Reply:
Think it might not make much sense to do this now that I'm looking at it again. We probably won't get much out of it and will probably add a lot of complexity. Maybe worth looking at it again later but I'll probably just remove the TODO for now.